postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -72,6 +75,9 @@ class DBExplainError(Enum):
     # there could be a problem with the EXPLAIN function (missing, invalid permissions, or an incorrect definition)
     failed_function = 'failed_function'
 
+    # a truncated statement can't be explained
+    statement_truncated = "statement_truncated"
```

Comment:
We could include the configured limit in the reason here to aid troubleshooting

Reply:
Great idea!